Source/CLI/Help.cpp

```diff
@@ -82,7 +86,11 @@ int Help()
     TEXTOUT("");
     TEXTOUT("--Language=raw");
     TEXTOUT("                    Display non-translated unique identifiers (internal text)");
-    TEXTOUT("--Details=1");
+    TEXTOUT("--LegacyStreamDisplay=1");
+    TEXTOUT("                    Display backward compatibility information for HD/HE audio and 3D/HDR video streams");
+    TEXTOUT("--Cover_Data=base64");
+    TEXTOUT("                    Include cover images as base64 in HTML, XML and JSON output");
+    TEXTOUT("--Trace_Level=1");

```

Comment:
```suggestion
    TEXTOUT("--Trace_Level=1, --Details=1");
```

For the moment I write a lot "Details=1", so let's keep it as a synonym